regulations/tests/views_comment_tests.py

```diff
@@ -0,0 +1,48 @@
+import json
+import mock
+
+from django.conf import settings
+from django.test import SimpleTestCase, override_settings
+
+
+@override_settings(
+    BUCKET='test-bucket',
+    ACCESS_KEY_ID='test-access-key',
+    SECRET_ACCESS_KEY='test-secret-key',
+    ATTACHMENT_MAX_SIZE=42,
+)
+class TestUploadProxy(SimpleTestCase):
+
+    @mock.patch('regulations.views.comment.boto3.Session')
+    @mock.patch('regulations.views.comment.get_random_string')
+    def test_get_url(self, get_random, session):
+        client = session.return_value.client
+        generate_presigned = client.return_value.generate_presigned_url
+        generate_presigned.return_value = 'test-url'
+        get_random.return_value = 'not-so-random'
+        resp = self.client.get('/comments/attachment?size=42')
+        self.assertEqual(resp.status_code, 200)
+        body = json.loads(resp.content.decode())
+        generate_presigned.assert_called_with(
+            ClientMethod='put_object',
+            Params={
+                'ContentLength': 42,
+                'ContentType': 'application/octet-stream',
+                'Bucket': settings.BUCKET,
```

Comment:
it'd be a tad clearer to use the constant defined above, 'test-bucket', the same way you're using 42.